manpy/simulation/CoreObject.py

```diff
@@ -561,11 +566,19 @@ def findReceiversFor(activeObject):
     # =======================================================================
     # signal the successor that the object can dispose an entity
     # =======================================================================
-    def signalReceiver(self):
+    def signalReceiver(self, transmitter=None):
+
         possibleReceivers = self.findReceiversFor(self)
```

Comment:
Are you sure that simply replacing this line with 
```python
         possibleReceivers = [transmitter] if transmitter else self.findReceiversFor(self)
```
is not enough?

Reply:
it is.